doc/api/assert.md

```diff
@@ -664,6 +663,7 @@ property set equal to the value of the `message` parameter. If the `message`
 parameter is `undefined`, a default error message is assigned. If the `message`
 parameter is an instance of an [`Error`][] then it will be thrown instead of the
 `AssertionError`.
+If no arguments are passed in at all `message` will be set to
```

Comment:
Incomplete sentence?

Reply:
Uh, somehow I did not finish it... solved.